website/docs/patterns/disabled-patterns/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,74 @@
+### Permissions 
+
+When a user does not have CRUD (Create, Read, Update and Delete) permissions, hide the related actions.
+
+In this example, the user's role is **viewer** and they cannot **deploy** or **destroy** a cluster. An explanation is not needed as users understand that their account has limited permissions and that they require an additional user's help to gain these permissions (often someone with admin or similar roles).
+
+![Image showing two dropdown with lists open, one with only one menu item that is reader specific, while the other has more options that allow for further capabilities.](/assets/patterns/disabled-patterns/permissions-example.png)
+
+In scenarios where a user lands on a page they cannot access, provide a reason why with guidance so that users understand what steps they can take to rectify the error. So that users don't feel stuck, provide them a call to action to facilitate a path forward.
+
+![Showing a 403 error page with information about what is going on, with a way to go back and a way to log in with a different token](/assets/patterns/disabled-patterns/permissions-example-3.png)
+
+### Conditional availability
+
+When a feature or function is conditionally available depending on other steps that must be taken, provide context in place of that action so that users are able to take those steps to enable this feature.
+
+!!! Dont
+
+In this example, the button is disabled and no context is provided, leaving the user confused and unaware of the steps required to enable the feature.
+
+![Showing an interface with a disabled button that doesn't describe why that button is disabled. This is not a recommended experience.](/assets/patterns/disabled-patterns/conditional-availability-example-disabled.png)
+
+Disabled elements lack interactive states, preventing a user from hovering, focusing, or clicking them. For users with assistive technology (e.g., a screen reader), disabled elements are almost entirely ignored, including the text within.
+
+!!!
+
+
+!!! Do
+
+Instead, we recommended replacing the disabled element with contextual guidance to help users to understand why this feature isn't available, and providing them with clear steps to enable it.
+
+![Showing an interface with a compact alert explaining why the expected behavior isn't there with guidance.](/assets/patterns/disabled-patterns/conditional-availability-example-explanation.png)
+
+If possible, provide CTAs directly to the place where users can take immediate action.
+
+!!!
+
+
+### Upgrading
+
+When premium features are available and a user has the necessary permissions, highlighting a path to upgrade provides a graceful marketing opportunity.
```

Comment:
**question:** Where did the term "premium" come from? It's not in the original guidance, and I don't think I've ever heard of us offering "premium" features before.

Reply:
The term was used specifically to indicate anything that requires additional payment. Is there a word you prefer to use? Maybe "When upgraded features..."?

Reply:
I updated wording to say "upgraded feature"